pyanaconda/payload/__init__.py

```diff
@@ -38,6 +38,7 @@
 from pyanaconda.image import opticalInstallMedia, verifyMedia, verify_valid_installtree
 from pyanaconda.core.util import ProxyString, ProxyStringError
 from pyanaconda.core.regexes import VERSION_DIGITS
+from pyanaconda.payload import storage as payload_storage
```

Comment:
Please, move the functions to `pyanaconda.payload.utils`, this is too confusing.

Reply:
Will do.